src/ReleaseHistory.md

```diff
@@ -382,7 +382,7 @@
 * BUGFIX: Multitool crashes on launch: Can't find CommandLine.dll. https://github.com/microsoft/sarif-sdk/issues/1487
 
 ## **v2.1.2** [Sdk](https://www.nuget.org/packages/Sarif.Sdk/2.1.2) | [Driver](https://www.nuget.org/packages/Sarif.Driver/2.1.2) | [Converters](https://www.nuget.org/packages/Sarif.Converters/2.1.2) | [Multitool](https://www.nuget.org/packages/Sarif.Multitool/2.1.2)
-* API BREAKING: Change location.logicalLocation to logicalLocations array. https://github.com/oasis-tcs/sarif-spec/issues/414
+* API BREAKING: Change result.logicalLocation to logicalLocations array. https://github.com/oasis-tcs/sarif-spec/issues/414
 
 ## **v2.1.3** [Sdk](https://www.nuget.org/packages/Sarif.Sdk/2.1.3) | [Driver](https://www.nuget.org/packages/Sarif.Driver/2.1.3) | [Converters](https://www.nuget.org/packages/Sarif.Converters/2.1.3) | [Multitool](https://www.nuget.org/packages/Sarif.Multitool/2.1.3)
```

Comment:
>Visual Studio log [](start = 50, length = 17)

Should we call out that its MSBuild log file ? (VS logs could mean something more generic?)

Reply:
*sigh* I did struggle with the naming. I supposed MSBuild is better. Even that's not perfect because _any_ tool or script that produces error lines in this standard format will work. I'll merge this but will ask for a call from Michael.

---
In reply to: [294517116](https://github.com/microsoft/sarif-sdk/pull/1533#discussion_r294517116) [](ancestors = 294517116)